src/storage/storage-async.ts

```diff
@@ -200,6 +220,18 @@ export class StorageAsync implements IStorageAsync {
         return docs[0];
     }
 
+    /** Returns a collection of docs for a given query.
```

Comment:
More accurate to say "returns an array" instead of "returns a collection"

Reply:
Agreed, updated.